docker/install/centos_build_llvm.sh

```diff
@@ -7,7 +7,7 @@ LLVM_VERSION_MAJOR=$1
 source /multibuild/manylinux_utils.sh
 
 detect_llvm_version() {
-  curl -sL "https://api.github.com/repos/llvm/llvm-project/releases?per_page=50" | \
+  curl -sL "https://api.github.com/repos/llvm/llvm-project/releases?per_page=100" | \
```

Comment:
This is bound to break again eventually, would it be more stable to just use the actual link here instead of trying to detect it automatically?

Reply:
Merging this as a hotfix since we need to rebuild the images for a release but it's probably worthwhile to make sure everything is version pinned in a follow up

Reply:
I agree we should pin. I just didn't want to make both changes in a single PR without chatting with anyone.